src/message-stream.ts

```diff
@@ -211,6 +217,28 @@ export class MessageStream extends PassThrough {
       this.destroy(e);
     }
   }
+  /**
+   * gax streams are basically just proxy streams that allow grpc streams to be
+   * created and set asychronously without the user having to deal with that.
+   * However since there isn't a way to specify the flow control limits on any
+   * grpc stream or tap into an event when its been created, we'll stub the
+   * setter method to adjust the highWaterMark before data starts flowing.
+   *
+   * @private
+   *
+   * @param {stream} stream Duplexify stream.
+   */
+  private _interceptGrpcStream(stream: GaxDuplex): void {
+    const setReadable = stream.setReadable;
```

Comment:
Would this break if GAX decides to replace Duplexify, or if Duplexify itself changes their API and GAX upgrades?

Reply:
Yes, absolutely. Seems unlikely that Duplexify would change, but if gax were too, I would think it would not come as a patch release and should be caught much earlier on.

I admit this isn't a great way to accomplish what I need here, but the only other option available for the desired effect would be to bypass gax entirely and create my own grpc client.